documentation/manifest.md

```diff
@@ -1,10 +1,10 @@
 # First Launch
 - **Start page is displayed by default on first launch**
   - **Click on GitHub link on start page under Open > Checkout from**
-    - [ ] Team Explorer Window exposed
-    - [ ] Clone a GitHub Repository dialog visible
+    - [ ] Team Explorer Window exposed (only appears when clone dialog is closed)
+    - [x] Clone a GitHub Repository dialog visible
   - **If last solution was in a git repo hosted on GitHub**
-    - **Team Explorer Home page shows:**
+    - **Team Explorer Home page shows:** (VS 2017 doesn't seem to remember. Shows Connect page)
```

Comment:
I believe all of the bold text are just titles, so this part isn't a step. Basically we want to confirm all the checkboxes in this section (header/PR/Pulse are in the Home page cause that's our stuff)